packages/diagram_capture/lib/theme.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'diagram_capture.dart';
+
+/// The size of the viewport that diagrams are captured in.
+const Size kDefaultDiagramViewportSize = Size(1280.0, 1024.0);
+
+/// The default [MediaQuery] for diagrams.
+///
+/// This has no padding or insets, a devicePixelRatio of 1.0, and a
+/// textScaleFactor of 1.0.
+final MediaQueryData kDefaultMediaQuery = MediaQueryData(
+  size: kDefaultDiagramViewportSize,
+  devicePixelRatio: DiagramFlutterBinding.instance.pixelRatio,
+);
+
+/// A [MaterialApp] with defaults that are appropriate for capturing diagrams.
+///
+/// By default we emulate the theme of an Android device in light mode, and
+/// force the Roboto font.
```

Comment:
The  API docs website defaults to the "Open Sans" font, so it might match better than Roboto. I believe it has a similar license.